src/common/users.ts

```diff
@@ -180,10 +189,10 @@ export const getUserReadingRank = async (
       return Promise.resolve(null);
     }
 
-    const start = new Date(startOfWeek(now).getTime()).toISOString();
-    const end = new Date(endOfWeek(now).getTime()).toISOString();
+    const start = now.toISOString();
+    const end = start;
```

Comment:
Since these are now the same we might want to change the naming to `timezone` I think this will become very confusing in the future.
What do you think?

Subnote about the above one if it doesn't break the profile implementation.